static_src/test/functional/expressive_empty_state.spec.js

```diff
@@ -0,0 +1,40 @@
+
+import http from 'axios';
+import dedent from 'dedent';
+
+import EmptyElement from './pageobjects/empty.element';
+import OrgQuicklookElement from './pageobjects/org_quicklook.element';
+
+describe('Expressive empty state', function () {
+  let appsEmptyState;
+
+  it('navigates to page', function () {
+    browser.url('/');
+  });
+
+  it('changes to app emtpy state', function(done) {
```

Comment:
You're not calling this `done` callback which I think is throwing some things off.